blog/incubation-announcement.md

```diff
@@ -0,0 +1,74 @@
+---
+title: OpenFGA Accepted into CNCF Incubation 🎉
+description: OpenFGA has been accepted into the Cloud Native Computing Foundation Incubation stage
+slug: incubation-announcement
+date: 2025-11-18
+authors: [aaguiar, jakub]
+tags: [announcement]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# OpenFGA Accepted into CNCF Incubation 🎉
+
+OpenFGA has been accepted into the Cloud Native Computing Foundation Incubation stage. This milestone reflects years of community work and real-world adoption.
```

Comment:
Let's show some excitement here.

"We are super proud" "We are thrilled to announce",.